internal/consumer/consumer.go

```diff
@@ -183,6 +183,7 @@ func (i *InventoryConsumer) Consume() error {
 
 				resp, err = i.ProcessMessage(headers, relationsEnabled, e)
 				if err != nil {
+					Incr(i.MetricsCollector.msgProcessFailures, "ProcessMessage", err)
```

Comment:
Double check me here, but i think i left this off because there were a few places where ProcessMessage could fail inside that function so instead of doing it at the top level, i only did it inside the fuction so that i could point out the exact function inside process message that failed. I think this would increment it a second time on each failure?

Reply:
Yup, you're spot on it would increment twice as each, as each create, update, and delete already have a `Incr()` call.

Reply:
Gonna go ahead and fix that, thanks!